mitodl/mitxpro
# Upgrade sentry for Python and JS

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #646 
Fixes #660 

#### What's this PR do?
Upgrades sentry client for Python and Javascript. Everything should work as before.

#### How should this be manually tested?
Make sure that errors are still logged as before. I would configure the PR build with the same DSN as the CI build, then try to create some exceptions for celery, the python shell, and via javascript. 

Python:
 - From the docs, run something like this in the shell: `sentry_sdk.capture_exception(Exception("This is an example of an error message."))`.

Celery:
 - I ran a task with an invalid value: `sync_product_with_hubspot.delay(-123)`

JS:
 - I'm not sure here but I managed to cause an error unintentially in the registration page: https://sentry.io/organizations/mit-office-of-digital-learning/issues/1099757481/?project=1413655
